update css

mistermaxis-zz opened the pull request.

mistermaxis-zz committed:
update css

kiranitor123 commented:
install weblint, stylelint
Add Readme
Create better commits
Improve the title and description of the pull request
header is missing, h1
Add gitignore
Remove test.md

mistermaxis-zz committed:
Create .gitignore

Created the ignore file to include test.md

mistermaxis-zz committed:
Create README.md file

Created a README file with a basic description of the repository's contents.

mistermaxis-zz committed:
Change h1 to header

Changed the tag so it's a header tag instead of an h1 tag and also the css file to match.

mistermaxis-zz committed:
Installed webhint and styleint

mistermaxis-zz committed:
Fixed error in name of styleint file
